assets/overhead_test/config.yaml

```
@@ -0,0 +1,45 @@
+  # Query all JSON log lines containing "attr" object with a nested "message" object with a "msg"
+  # field that starts with "log_release" and a "session_name" field that is "connection"
+  - '"attr.message.msg: log_release* AND attr.message.session_name: connection"'
+  # Query all JSON log lines where "ctx" is "initandlisten" and either "msg" does not equal to
+  # "WiredTiger message" or "attr.message.msg" starts with "log_remove"
+  - '''ctx: initandlisten AND (NOT msg: "WiredTiger message" OR attr.message.msg: log_remove*)'''
+  # Query all JSON log lines where "c" is "WTWRTLOG" and "attr.message.ts_sec" is greater than
+  # 1679490000
+  - '"c: WTWRTLOG AND attr.message.ts_sec > 1679490000"'
+  # Query all JSON log lines where "ctx" is "FlowControlRefresher" and "attr.numTrimmed" equals 0
+  - '"ctx: FlowControlRefresher AND attr.numTrimmed: 0"'
+
```

YAMLlint (1.37.1)

[error] too many blank lines (1 > 0)

(empty-lines)